admin/.adminjs/entry.js

```diff
@@ -0,0 +1 @@
+AdminJS.UserComponents = {}
```

Comment:
Why is this needed? I'd say overall `.adminjs` folder should not be commited, but I am not sure at all